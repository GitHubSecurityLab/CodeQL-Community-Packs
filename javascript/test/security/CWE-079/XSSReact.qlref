security/CWE-079/XSSReact.ql
