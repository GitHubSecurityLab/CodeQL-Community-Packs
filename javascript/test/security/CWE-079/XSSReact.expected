edges
| app.jsx:12:11:12:27 | [query, setQuery] | app.jsx:12:11:16:6 | query | provenance |  |
| app.jsx:12:11:16:6 | query | app.jsx:17:49:17:53 | query | provenance |  |
| app.jsx:12:31:16:6 | useQuer ... \\n    }) | app.jsx:12:11:12:27 | [query, setQuery] | provenance |  |
| app.jsx:17:11:17:45 | { x: nu ... lters } | app.jsx:17:11:17:53 | searchQuery | provenance |  |
| app.jsx:17:11:17:53 | searchQuery | app.jsx:26:52:26:62 | searchQuery | provenance |  |
| app.jsx:17:49:17:53 | query | app.jsx:17:11:17:45 | { x: nu ... lters } | provenance |  |
nodes
| app.jsx:12:11:12:27 | [query, setQuery] | semmle.label | [query, setQuery] |
| app.jsx:12:11:16:6 | query | semmle.label | query |
| app.jsx:12:31:16:6 | useQuer ... \\n    }) | semmle.label | useQuer ... \\n    }) |
| app.jsx:17:11:17:45 | { x: nu ... lters } | semmle.label | { x: nu ... lters } |
| app.jsx:17:11:17:53 | searchQuery | semmle.label | searchQuery |
| app.jsx:17:49:17:53 | query | semmle.label | query |
| app.jsx:26:52:26:62 | searchQuery | semmle.label | searchQuery |
subpaths
#select
| app.jsx:26:52:26:62 | searchQuery | app.jsx:12:31:16:6 | useQuer ... \\n    }) | app.jsx:26:52:26:62 | searchQuery | Cross-site scripting vulnerability due to $@. | app.jsx:12:31:16:6 | useQuer ... \\n    }) | user-provided value |
